sw/host/opentitantool/src/command/image.rs

```diff
@@ -183,6 +186,7 @@ impl CommandDispatch for ManifestUpdateCommand {
     ) -> Result<Option<Box<dyn erased_serde::Serialize>>> {
         let mut image = image::Image::read_from_file(&self.image)?;
         let mut update_length = self.update_length;
+        let private_keys_sign = self.private_keys_sign;
```

Comment:
You don't need this `let` binding; you can access this struct field directly in the `if` statement below like `if self.private_keys_sign { ... }`